File: BackendAPI/Application/DTOs/Output/KoperOutputDto.cs
namespace Application.DTOs.Output;

public class KoperOutputDto
{
    public Domain.Enums.AccountType AccountType { get; init; } = Domain.Enums.AccountType.Koper;
    public required string Email { get; set; } = string.Empty;
    public required string FirstName { get; set; } = string.Empty;
    public required string LastName { get; set; } = string.Empty;
    public required string Telephone { get; set; } = string.Empty;

    public required int PrimaryAddressId { get; set; }
    public required List<AddressOutputDto> Addresses { get; set; } = new();
}

File: FrontendWeb/src/declarations/dtos/output/KoperOutputDto.ts
import { AccountType } from '../../enums/AccountTypes';
import { AddressOutputDto } from './AddressOutputDto';

// KoperOutputDto.ts
export interface KoperOutputDto {
	accountType: AccountType;
	email: string;
	firstName: string;
	lastName: string;
	telephone: string;
	primaryAddressId: number;
	addresses: AddressOutputDto[];
}
